Business Hub - mWeb / Desktop - Marquee

jsandland opened the pull request:
Describe your specific features or fixes:
* Remove dividing line at bottom of marquee on mobile
* Logo above padding
* Content Toggle padding
* Ensure text wraps on smaller viewports
* Header size on mWeb and Desktop
* Content Toggle font-size on mWeb and Desktop

**No Authoring Changes**
Authoring Example: [business-hub-marquee.docx](https://adobe.sharepoint.com/:w:/r/sites/adobecom/Express/website/drafts/jsandlan/business-hub-marquee.docx?d=wf0427444e3bf4f2ebb31fb129622e0fe&csf=1&web=1&e=FBNlXx)

**mWeb**:
Resolves: [MWPW-168284](https://jira.corp.adobe.com/browse/MWPW-168284)
**Desktop**:
Resolves: [MWPW-168264](https://jira.corp.adobe.com/browse/MWPW-168264)


Test URLs:
- Pre Migration Link: https://adobe.com/express/
- Before: https://www.stage.adobe.com/express/business
- After: https://business-hub-marquee--express-milo--adobecom.aem.page/drafts/jsandlan/business-hub-marquee

JingleH reviewed (dismissed):
do we know why grid-marquee is also modified in this PR? I don't see the request of updating it in the ticket and it is affecting the h1 sizes of homepage as well...

jsandland commented:
> do we know why grid-marquee is also modified in this PR? I don't see the request of updating it in the ticket and it is affecting the h1 sizes of homepage as well...

I've replaced the grid-marquee with the headline.
